mne_bids/read.py

```diff
@@ -191,6 +192,12 @@ def read_raw_bids(bids_fname, bids_root, verbose=True):
         Full name of the data file
     bids_root : str
         Path to root of the BIDS folder
+    allow_maxshield : bool | str (default False)
+        If True, allow loading of data that has been recorded with internal 
+        active compensation (MaxShield). Data recorded with MaxShield should 
+        generally not be loaded directly, but should first be processed 
+        using SSS/tSSS to remove the compensation signals that may also affect 
+        brain activity. Can also be “yes” to load without eliciting a warning.
```

Comment:
I don't think there is any need to expose these param. 

You can simply copy what was passed to `raw` by using `raw._init_kwargs`

Reply:
Ok, I was not aware of this option and spent quite some time figuring out what to do. Still, I think this might be a common error and a very discouraging one for newbies. Can it be documented somewhere or integrated in the error message?

Reply:
actually it's my bad, sorry. Somehow I thought you were modifying the writing function. This looks good to me. Do you want to update `whats_new.rst`?

Reply:
ok, I think I did that. how about the failed check?